Makefile

```diff
@@ -2,7 +2,7 @@ SUBDIRS = factory-package-news abichecker
 
 include Makefile.common
 
-pkgdata_BINS = $(shell find * -maxdepth 0 -executable -type f)
+pkgdata_BINS = $(shell find * -maxdepth 0 -executable -type f|sed '/.*rb$$/d')
```

Comment:
If you want to exclude something, just drop the executable bit or if that's not possible, add a `! -name`